src/customer/strategies/braintree/braintree-visacheckout-customer-strategy.ts

```diff
@@ -121,10 +124,22 @@ export default class BraintreeVisaCheckoutCustomerStrategy implements CustomerSt
                     state.shippingAddress.getShippingAddress(),
                     state.billingAddress.getBillingAddress()
                 )
-                .then(() => this._store.dispatch(this._checkoutActionCreator.loadCurrentCheckout()));
+                .then(() => {
+                    this._store.dispatch(this._checkoutActionCreator.loadCurrentCheckout());
```

Comment:
Could we add async like below?  
```suggestion
                .then(async () => {
                    await this._store.dispatch(this._checkoutActionCreator.loadCurrentCheckout());
```